content/doc/book/pipeline/syntax.adoc

```diff
@@ -807,8 +807,9 @@ provide when triggering the Pipeline. The values for these user-specified
 parameters are made available to Pipeline steps via the `params` object,
 see the <<parameters-example>> for its specific usage.
 
-Each parameter has a _Name_ and some sort of _Value_, depending on the parameter type.
-These name-value pairs will be exported as environment variables when the build starts, allowing subsequent parts of the build configuration (such as build steps) to access those values, e.g. by using the `+{PARAMETER_NAME}+` syntax (or `%PARAMETER_NAME%` on Windows).
+Each parameter has a _Name_ and  _Value_, which depends on the parameter type.
+These name-value pairs are exported as environment variables when the build starts, allowing subsequent parts of the build configuration, such as build steps, to access those values.
+For example, this can be performed by using the `+{PARAMETER_NAME}+` syntax (or `%PARAMETER_NAME%` on Windows).
```

Comment:
```suggestion
Each parameter has a _Name_ and _Value_, depending on the parameter type.
This information is exported as environment variables when the build starts, allowing subsequent parts of the build configuration to access those values.
For example, this can be performed by using the `+{PARAMETER_NAME}+` syntax (or `%PARAMETER_NAME%` on Windows).
```
I found an extra space, and wanted to suggest something other than name-value pairs for the parameters. If that does not convey the same message, I would be happy to discuss other options for that wording.

Reply:
Thanks @kmartens27 for the suggestion.I have used the words _Name_ and _Value_ because they were stated in [here](https://github.com/jenkinsci/jenkins/blob/8a29e8858b49da45c0071f690d4ceebe8db2f18f/core/src/main/resources/hudson/model/ParametersDefinitionProperty/help.html#L11-L15) however we can always discuss for other options to improve the readability. Do you have any suggestions?

Reply:
Right, sorry about that!
Based on that, the name-value pairs should be fine. Disregard that part of the suggestion! Thanks very much @Vandit1604